src/middlewares/commentAccess.js

```diff
@@ -0,0 +1,25 @@
+import { users, tripRequests } from '../database/models';
+import Customize from '../helpers/Customize';
+
+const commentAccess = async (req, res, next) => {
+  const { id } = req.user;
+  const { tripRequestId } = req.params;
+  const isManager = await users.findOne({
+    where: {
+      id,
+      roleId: 6,
+    }
+  });
+  const isRequester = await tripRequests.findOne({
+    where: {
+      id: tripRequestId,
+      userId: id,
+    }
+  });
```

Comment:
I suggest moving this query into another helper file so that everyone who need will be able to use it.